data/battle_scripts_1.s

```diff
@@ -442,6 +442,21 @@ gBattleScriptsForMoveEffects::
 	.4byte BattleScript_EffectMaxMove                 @ EFFECT_MAX_MOVE
 	.4byte BattleScript_EffectGlaiveRush              @ EFFECT_GLAIVE_RUSH
 	.4byte BattleScript_EffectBrickBreak              @ EFFECT_RAGING_BULL
+	.4byte BattleScript_EffectFilletAway              @ EFFECT_FILLET_AWAY
+
+BattleScript_EffectFilletAway:
```

Comment:
The correct order of Fillet Away is first boost stats, then cut HP. Also, the current way you have it, the attack animation would still play even if the stats can't be lowered. I recommend keeping the jump to Shell Smash, and then at the end of the Shell Smash boosts, do something like `jumpifmoveeffect EFFECT_FILLETAWAY BattleScript_FilletAwayCutHP`.

Reply:
Is there a jumpifmoveeffect? If we have to do this then it'd be easier to just not jump to shell smash code anymore and do the stat changes here.

Reply:
By default I don't believe that battle script command exists, but it's not so hard to implement